### DISCUSSION TITLE
Can I specify path to maxima exectuable in a wxmaxima configuration file

### DISCUSSION THREAD

**Discussion by ejbarth** [wxMaxima-developers/wxmaxima#1725] [General]
Hi, 

 is there a way --- maybe in wxmaxima-init.mac or other configuation file --- to specify the path of the maxima executable? 
I'd like to avoid the user having to do this through wxMaxima Preferences menu  (in part because the Preference window that appears on my mac  has illegible tab names)

The longer version of the story:
 I'm working on an approach to macOS install:   a single shell script that has everything needed to install homebrew and subsequently wxmaxima and maxima.  The motivation is to make the installation process manageable for the many undergraduate students in my classes who use macOS on either Intel and Apple Silicon architectures.  Before the birth of Apple Silicon, I shared my experience working with student installation at my macOS installation page at themaximalist.org .  That got lots of hits, and I think was pretty successful.
Now, I'm looking for another approach and hoping a homebrew install can be made workable for non-expert mac users.

I've made some progress.  Here's the maxima and wxmaxima info I see in the  current homebrew installation
build_info();
build_info(version="5.46.0",timestamp="2022-11-01 06:30:28",host="x86_64-apple-darwin21.6.0",lisp_name
="SBCL",lisp_version="2.2.10",maxima_userdir="/Users/barth/.maxima",maxima_tempdir=
"/var/folders/z7/dltdx1px7pv09knc8z13cxrjwxpg9q/T",maxima_objdir=
"/Users/barth/.maxima/binary/5_46_0/sbcl/2_2_10",maxima_frontend="wxMaxima",maxima_frontend_version=
"22.09.0_MAC")

Thank you for your help and consideration.
In my view the macOS issue is a key obstacle that prevents wider use of Maxima (and wxMaxima).  I'm trying to make some small contribution in that regard.
Eric

**Comment by daute** [comment]
wxMaxima supports the command line option `--maxima=<path-to-maxima>` (or `-m <path-to-maxima>` which does the same).
